modules/gcp_bigquery_view/main.tf

```diff
@@ -0,0 +1,13 @@
+resource "google_bigquery_table" "view" {
+  #checkov:skip=CKV_GCP_121:Deletion protection is not needed for views.
+  #checkov:skip=CKV_GCP_80:Encryption at rest does not apply to views.
+  dataset_id          = var.dataset_id
+  table_id            = var.name
+  deletion_protection = false
```

Comment:
deletion_protection should be a input variable.

Reply:
Why? Views do not contain any data so there is no data loss if the view gets dropped.

Reply:
It is not about the data in views. It is about if the view can be deleted without any restrictions.